### ISSUE TITLE
Cannot open links

### ISSUE THREAD

**Issue by FuriousLlama** [StarlaneStudios/vscode-comment-anchors#212]
I tried different ways to link to a file but I don't see the highlighted text when hovering over a LINK path. I've tried disabling all other extensions but that did not help. 

Notice how a regular URL does have the expected underline while the other links do not.

![image](https://github.com/StarlaneStudios/vscode-comment-anchors/assets/12201292/887a45f0-ca6a-4164-a8ac-1daf185b4ef3)

**Comment by Challanger524** [comment]
Same thing with filepaths without `:<num>` and `#<anchor>` at the end, only workaround is to append `:0` or `:1` to the filepath like: `// LINK readme.md:0`

**Comment by caiobolzani** [comment]
First, enable links in the editor, so any http regular link will be clickable...
then I used the following syntax (Windows OS):
// LINK  \Users\MyName\Desktop\file.txt and it becomes clickable also (no C:, no double backslash)
hope it helps

**Comment by talkingwires** [comment]
Nope. I have a similar issue. Regular URL links work, Comment Anchor links do not. I have tried every permutation I can think of:

`{/**LINK Centered.astro **/}`
`{/**LINK ./Centered.astro **/}`
`{/**LINK /src/layoutsCentered.astro **/}`
`{/**LINK ./src/layoutsCentered.astro **/}`
`{/**LINK \\wsl.localhost\Ubuntu-24.04\home\USER\projects\hypna\src\layouts\Centered.astro **/}`
`{/**LINK ~\projects\src\layouts\Centered.astro **/}`
`{/**LINK \Users\MyName\literally_any_other_path.file **/}`

And, as per @Challanger524 

`{/**LINK Centered.astro:0 **/}`
`{/**LINK ./Centered.astro:0 **/}`
…and so forth.

Out of exasperation, I created a blank HTML file within my Windows User directory and tried linking to another file in the same directory:

`<!--LINK \Users\MyName\Projects\scraper.py -->`

Nope. And then, on a hunch, I created a blank CPP file and formatted the comment accordingly:

`//LINK \Users\MyName\Projects\scraper.py`

And it worked! Something about the way Comment Anchors process syntax and determines what is and is not a comment is borked. Perhaps I will investigate further later, but for now, I’m tired of looking at it and need to get back to work.

**Comment by Challanger524** [comment]
Originally I searched for link opener when VSCode's was broken on Windows (it is fixed now)
